plans/vnets.tf

```diff
@@ -59,131 +59,123 @@ resource "azurerm_virtual_network" "public_prod" {
 
 resource "azurerm_subnet" "public_dmz" {
   name                      = "dmz-tier"
-  resource_group_name       = "${azurerm_resource_group.public_prod.name}"
-  virtual_network_name      = "${azurerm_virtual_network.public_prod.name}"
+  resource_group_name       = azurerm_resource_group.public_prod.name
+  virtual_network_name      = azurerm_virtual_network.public_prod.name
   address_prefix            = "10.0.99.0/24"
-  network_security_group_id = "${azurerm_network_security_group.public_dmz_tier.id}"
```

Comment:
last time I was looking at this you needed both for now, until 2.0 is out, otherwise terraform keeps seeing a change in the diff unless you suppress changes on the subnet

Reply:
Indeed, you are right, I also remember having that issue, but this time I fixed all warning reported by terraform plan, now i am running a terraform apply, to validate that everything is working correctly.